crates/stdsimd-verify/tests/x86-intel.rs

```diff
@@ -240,8 +240,8 @@ fn matches(rust: &Function, intel: &Intrinsic) -> Result<(), String> {
     // ensuring that we've actually enabled the right instruction
     // set for this intrinsic.
     match rust.name {
-        "_bswap" => {}
-        "_bswap64" => {}
+        "_bswap" | "_bswap64" | "_addcarry_u32" | "_addcarry_u64" | "_subborrow_u32"
+        | "_subborrow_u64" => {}
```

Comment:
Might want to add a comment here stating that we skip these intrinsics because they don't need / have any target features in the spec.

Reply:
Although some of them do have the `adx` target feature =/

Reply:
Can do! Note that these aren't actually connected to adx apparently, only the ones with "x" in the name are related to adx. (not entirely sure what's going on here myself...)